packages/audio/alsa-utils/package.mk

```diff
@@ -26,20 +26,8 @@ post_configure_target() {
 }
 
 post_makeinstall_target() {
-  rm -rf ${INSTALL}/lib ${INSTALL}/var
-  rm -rf ${INSTALL}/usr/share/alsa/speaker-test
-  rm -rf ${INSTALL}/usr/share/sounds
```

Comment:
They're used by `speaker-test`, which can be useful when testing your card.  For instance, when trying to get the driver for my DAC board working, I had to look for and download wav files from the Internet in oder to test it.  It would have been convenient, if they were already available.
The wav files are not absolutely necessary, as `speaker-test` can work without them, either by sending noise to the card, or user-specified wavs, but these files are the default ones used with the `-t` switch, so they're necessary if we don't want `speaker-test` to appear broken.  They're also useful for testing speaker connections i.e. whether the left-right speakers haven't been connected the other way round etc.
So I would argue that they're useful as debugging tools that a desktop user might already be familiar with and expect to find.  I don't see how they hurt either, except for taking up space.  If using the least possible space is considered important, they can be skipped.

Reply:
It would be preferrable to include a single .wav sample (a reduced-in-size one if possible; doesn't need to be one from the original package) and then create symlinks that recreate the appearance of having the standard files, but avoiding the bloat of actually including them. It might sound daft, but that's the ethos of a minimalist distro :)